### DISCUSSION TITLE
Frequent message about reviewing connection security

### DISCUSSION THREAD

**Discussion by writerwrenee** [GorvGoyl/ChatGPT-Writer#8] [Report Bugs and Issues]
![chat openai com_security](https://user-images.githubusercontent.com/36651763/210411790-f799fd0c-61c7-45ba-b4c7-5c3b1441fb2c.png)

I receive this message every time I try to use the extension, even if it's only been five or 10 minutes. I'm using Chrome on a MacBook Pro. Are there any specific browser settings I should try adjusting? 

Thanks, 
Renee

**Comment by GorvGoyl** [comment]
there's not much you can do.. it's a restriction from the chat.openai.com

**Comment by Dezzign** [comment]
Hmm, I get this too even straight after I've logged in 10 seconds ago. I'm using the Brave browser.
I can't get rid of that message no matter what I do...

**Reply by GorvGoyl** [reply]
for brave browser, follow these steps https://github.com/GorvGoyl/ChatGPT-Writer/discussions/2#discussioncomment-4581141

**Reply by Dezzign** [reply]
Perfect, that worked nicely. Had to also relaunch Brave after changing it. Thanks :)

**Comment by GorvGoyl** [comment]
@writerwrenee if you're using brave browser then follow these steps https://github.com/GorvGoyl/ChatGPT-Writer/discussions/2#discussioncomment-4581141

**Comment by brianitsup** [comment]
Yep, issue persists for me too for Chrome on Windows 11.

**Comment by anewbies** [comment]
Se estiver usando o Google Chrome, entre na configuração chrome://settings/syncSetup e marque a caixa on "verificação de ortografia aprimorada" e o problema estará resolvido.
![image](https://user-images.githubusercontent.com/14927641/213434388-b31201f0-4ac8-4fcc-a0eb-626e83ff2ef3.png)
